lib/fastlane/plugin/firebase_test_lab/helper/ios_validator.rb

```diff
@@ -21,7 +21,17 @@ def self.validate_ios_app(file_path)
             unless size == 1
               UI.user_error!("The app bundle may contain only one scheme, #{size} found")
             end
-            _, scheme_conf = conf.first
+            
+            scheme_conf = nil
+            conf.each_key { |key| 
+              if key != '__xctestrun_metadata__' 
+                scheme_conf = conf[key] 
+              end 
```

Comment:
We can move the line 22 logic here:
```ruby
if scheme_conf.nil?
  scheme_conf = value
else
  UI.user_error!("The app bundle may contain only one scheme.")
end

Reply:
Yes! Nice. I've force-pushed an update. Check it out.

I've used your suggestion, however in the body of the `if` we still need to check if the key is '__xctestrun_metadata__', otherwise that schema will be set first if it's encountered first. :)